app/components/ui/atoms/Textarea.js

```diff
@@ -45,14 +41,24 @@ class Textarea extends React.Component {
     this.inputId = `textarea-${Math.round(Math.random() * 1e12).toString(36)}`
   }
   render() {
-    const { label, value = '', readonly, ...props } = this.props
```

Comment:
Why did you remove default value for `value` here? I assume whoever added it, did it for a reason? Maybe somewhere this gets called without specifying value?

Reply:
I've added `value` as a required propType, to encourage people using this component by passing in `value` & `onChange` in such a way that this is a controlled React component.

To my mind, it wouldn't make any sense for someone to use `TextArea`, without passing in value dynamically. If it is a required value anyway, there's no need to set `value=''` by default.

Pubsweet's docs specify `initialState: { value = '' } in their styleguidist docs, in an effort to encourage people to overwrite it. See my own description in our markdown file for this component. I hope that this is a good explanation, that will encourage proper usage.

Reply:
I just checked and TextArea doesn't seem to be used without an initial value, so it's fine for now.

Regarding the other thing you mentioned, to me it seems quite the opposite. A TextArea that forces you to pass an empty value every time you need an empty one seems like a weird design decision (I imagine most of the time you'd use an empty text area, and only in few other cases you'd want something already in there).